src/main/java/frc/robot/subsystems/vision/VisionConstants.java

```diff
@@ -0,0 +1,8 @@
+package frc.robot.subsystems.vision;
+
+public class VisionConstants {
+    public static final double CAMERA_HEAIGHT = 0;
+    public static final double CAMERA_PITCH = 0;
+    public static final double UPPER_CONE_TARGET11_X_DISTANCE = 0;
+    public static final double UPPER_CONE_TARGET11_Y_DISTANCE = 0;
```

Comment:
1. What does `11` mean? Is it the specific cone node?
2. What does "distance" mean? Distance from what?
3. [IIUC](https://en.wiktionary.org/wiki/IIUC), this is the location of the retroreflective tape on the cone node. If so, you should use WPILib's [`Pose3d`](https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/math/geometry/Pose3d.html).

Reply:
The names are only temporary.
These constants are supposed to be the x, y coordinates of a specific cone node.
11 means the left node in the leftmost grid section.

Reply:
> These constants are supposed to be the x, y coordinates of a specific cone node.

What about the Z axis?
> 11 means the left node in the leftmost grid section.

This is not clear from the name. At least add documentation that explains the naming.